src/LiveComponent/doc/index.rst

```diff
@@ -931,14 +931,18 @@ a class:
     <!-- add multiple classes when loading -->
     <div data-loading="addClass(opacity-50 disabled)">...</div>
```

Comment:
Realizing after your modification we should not use "disabled" both in addClass and addAttribute, it does not help.. Could 
```suggestion
    <div data-loading="addClass(opacity-50 text-warning)">...</div>
```

Reply:
Missed a bit of message there, thank you @javiereguiluz for guessing the end of my sentence and fixing it ❤️